python/lsst/ts/mtaos/mtaos.py

```diff
@@ -489,12 +489,14 @@ async def do_runWEP(self, data):
             # in MTAOS runWEP command.
             await self.model.run_wep(
                 visit_id=self.visit_id_offset + data.visitId,
-                extra_id=self.visit_id_offset + data.extraId
-                if data.extraId > 0
-                else None,
-                config=yaml.safe_load(data.config)
-                if len(data.config) > 0
-                else self.wep_config,
+                extra_id=(
+                    self.visit_id_offset + data.extraId if data.extraId > 0 else None
+                ),
+                config=(
+                    yaml.safe_load(data.config)
+                    if len(data.config) > 0
+                    else self.wep_config
+                ),
```

Comment:
Is the just the format change by `black`?

Reply:
yes